### ISSUE TITLE
Problems running Mobius Perf tests

### ISSUE THREAD

**Issue by dwnichols** [Microsoft/Mobius#597]
Hi,

I'm looking to run the perf tests in localmode on my Windows 10 machine but am getting stuck on a few errors and hoping for some direction. I have fully built Mobius and the samples all run correctly without issue.

Here are a list of my relevant environment variables:

`SPARKCLR_EXT_JARS=C:\Code\Mobius\build\localmode\..\runtime\dependencies\spark-csv_2.10-1.4.0.jar,C:\Code\Mobius\build\localmode\..\runtime\dependencies\commons-csv-1.4.jar`

`SPARKCLR_HOME=C:\Code\Mobius\build\localmode\..\runtime`

`SPARK_HOME=C:\Code\Mobius\build\localmode\..\tools\spark-2.0.0-bin-hadoop2.6`

`JAVA_HOME=C:\Program Files\Java\jdk1.8.0_66`

`HADOOP_HOME=C:\Code\Mobius\build\localmode\..\tools\winutils`

The command line:

`C:\Code\Mobius\build\runtime\scripts\sparkclr-submit.cmd --jars %SPARKCLR_EXT_JARS% --exe SparkCLRPerf.exe C:\Code\Mobius\csharp\Perf\Microsoft.Spark.CSharp\bin\Debug C:\Tests 1 C:\Code\Mobius\csharp\Perf\Microsoft.Spark.CSharp\data\deletionbenchmarktestdata.csv`

[Here](https://gist.github.com/dwnichols/3b78b33534355b46ad6fe482f2047f6c/8786c342402f4de3d93e523e9e7b4c943a794c97) is a gist with the output of the run.

To summarize:

- `RunRDDLineCount` ran successfully
- `RunRDDMaxDeletionsByUser` ran successfully, but threw two exceptions during execution
- `RunDFMaxDeletionsByUser` failed referencing a poorly formed directory I didn't specify `java.net.URISyntaxException: Relative path in absolute URI: file:C:/Code/Mobius/csharp/Perf/Microsoft.Spark.CSharp/bin/Debug/spark-warehouse`
- `RunDFLineCount` did not run. 

Any help would be greatly appreciated.

**Comment by dwnichols** [comment]
I read through the `RunSample.cmd` file and found that it was specifying `--conf spark.sql.warehouse.dir=` within the command line it was generating. Adding that to mine fixed many of the problems. This is not reflected in the docs for localmode or the perf tests. 

An updated summary:

- `RunRDDLineCount` ran successfully
- `RunRDDMaxDeletionsByUser` ran successfully, but threw two exceptions during execution
- `RunDFMaxDeletionsByUser` ran successfully
- `RunDFLineCount` ran successfully

[Here](https://gist.github.com/dwnichols/3b78b33534355b46ad6fe482f2047f6c) is an update of the output.
